test/zdtm_ct.c

```diff
@@ -102,10 +102,9 @@ int main(int argc, char **argv)
 	/*
 	 * pidns is used to avoid conflicts
 	 * mntns is used to mount /proc
-	 * net is used to avoid conflicts of parasite sockets
```

Comment:
This isn't the only reason. We want to isolate network tests from each other. All our tcp tests use hard-coded port numbers and they will start failing with this change.

Reply:
If you want to run tests without network namespaces in your environment, you can introduce an environment variable for that.

Reply:
Ok, I didn't see any tcp test failures, but maybe because I ran them with test sharding enabled. I'll rework the comment then instead of this patch.